packages/rocketchat-lib/server/models/Messages.coffee

```diff
@@ -210,6 +210,12 @@ RocketChat.models.Messages = new class extends RocketChat.models._Base
 
 		return @findOne query
 
+	findOneBySlackTs: (slackTs) ->
```

Comment:
Needed the ability to look up a slack message w/ just the timestamp because the react message coming from slack didn't include the bot ID